### PR TITLE
Milestone: Tic Tac Toe Project 

### PR DESCRIPTION
**Pull request by shubhsk88**
# Tic Tac Toe Project

This PR address the implementation of the tic tac toe project which is provided as part of the Odin project. This project uses the logic of tic tac toe to play the game.

We used :
- Tailwind CSS
- Font Awesome
- Javascript

### CODE REVIEW

File: game.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  const chances = [
+    [0, 4, 8],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 3, 6],
+    [2, 4, 6],
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+  ];
```
Comment (maelfosso) [inline game.js]: That is not something related to only one player. 
All the player will use it.

You should move it out from there. 
Define a method like `checkWinning` and declare it into as constant in order for it to be used to determine the game status.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @shubhsk88 @flpfar 

Nice UI/UX :smiley_cat: Really original. The game is working like a charm!
However, it's not sufficient to move forward. For that, you need to update your code to satisfy the requirement

1. You must separate the Board data structure to the Board DOM. 

So that, 
- When the user click on the Board DOM cell, you update the Board data structure. And then **render** it.
- You use the Board data structure (array of 9 cells) to check the game status, decide whether or not the game should continue. According to the result Update the Game/Board DOM

By doing think like that you will meet some missed requirements

2. As indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/objects_requirement.png), You should have a `game`, `gameboard` objects

3- There should be a function with the only responsibility of rendering the contents of the gameboard array to the webpage, as indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/render_requirement.png),

4- You should have a function that that allows a player to 'mark' a spot on the board, as indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/mark_requirement.png)

5- As requested also [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/win_requirement.png), there should be a method(s) to check the game's winning conditions

Kindly make the suggested changes and submit a new code review request
Thanks

Mael FOSSO

**Review by fatymahmed — APPROVED** [review]
## Hi @shubhsk88 @flpfar,
The UI looks good 👏👏
Good job implementing the changes 🎉🎉

## Status: Approved ✔️

File: index.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+  const startListeners = () => {
+    for (let i = 0; i < boardItems.length; i += 1) {
+      boardItems[i].addEventListener('click', () => {
+        if (gameBoard.board[i] === '') {
+          if (gameBoard.getTurn() % 2 === 0) {
+            gameBoard.board[i] = 'X';
+          } else {
+            gameBoard.board[i] = 'O';
+          }
```
Comment (fatymahmed) [inline index.js:101]: [OPTIONAL]
- You can improve this part by implementing another method that takes the letter (`X` or `O`) and position `i` as arguments then marks that specific position on the gameboard.

**Pull request merged**